src/Gui/MenuManager.cpp

```diff
@@ -338,6 +340,31 @@ void MenuManager::setup(MenuItem* item, QMenu* menu) const
     }
 }
 
+void MenuManager::setupMenuBarCornerWidgets() const {
+    QMenuBar* menuBar = getMainWindow()->menuBar();
+    ParameterGrp::handle hGrp = WindowParameter::getDefaultParameter()->GetGroup("General");
+
+    //Right corner widget
+    if (hGrp->GetBool("WBSRightCorner", false)) { //add workbench selector to menubar right corner widget.
+        if (!qobject_cast<QComboBox*>(menuBar->cornerWidget(Qt::TopRightCorner)))

```

Comment:
Should include `QComboBox`

Reply:
QComboBox is already being used in the original code above. So it should already have that include?

Reply:
True but actually it's not. :)